src/WebSockHttpFlexServer.cpp

```diff
@@ -22,6 +22,7 @@
 #include <boost/make_unique.hpp>
 #include <cstdlib>
 #include <iostream>
+#include <fstream>
```

Comment:
do we need this?

Reply:
Without it, does not compile `‘std::ifstream inputFile’ has initializer …
…but incomplete type error`, i.e. https://github.com/eclipse/kuksa.val/blob/0dd7ab3c3a5a07c986b627d3f59c30027054bcee/src/WebSockHttpFlexServer.cpp#L1403
i

Maybe with the older jsoncons/jwt was included somewhere deeper in the hierarchy.... But here really is correct